test_rclcpp/test/test_intra_process.cpp

```diff
@@ -68,8 +72,8 @@ TEST(test_intra_process_within_one_node, nominal_usage) {
     // wait for the first callback
     {
       size_t i = 0;
-      while (counter == 0 && i < 10) {
-        printf("spin_node_once() - callback (1) expected - try %zu/10\n", i++);
+      while (counter == 0 && i < 2) {
```

Comment:
Did you reduce the iterations here because you expect better performance from intra-process?